doc/man7/RAND_DRBG.pod

```diff
@@ -0,0 +1,312 @@
+The <public> and <private> DRBG will detect this on their next generate
+call and reseed, pulling randomness from <master>.
+
+The last feature has been added to support the common practice used with
+previous OpenSSL versions to call RAND_add() before calling RAND_bytes().
+
+
+=head2 Entropy Input vs. Additional Data
+
+The DRBG distinguishes two different types of random input: I<entropy>,
+which comes from a trusted source, and I<additional input>',
+which can optionally be added by the user and is considered untrusted.
+It is possible to add I<additional input> not only during reseeding,
+but also for every generate request.
+This is in fact done automatically by L<RAND_DRBG_bytes(3)>.
+
+
+=head2 Configuring the Random Seed Source
+
+When configuring OpenSSL, the seed source can specified using the
+following option.
+
+   --with-rand-seed=seed1[,seed2,...]
+
+     os:          Use a trusted operating system entropy source (default)
+     getrandom:   Use the L<getrandom(2)> if available.
+     devrandom:   Use the /dev/urandom device.
+     egd:         Check for an entropy generating daemon.
+     rdcpu:       Use the RDSEED or RDRAND command if provided by the CPU.
+     librandom:   Use librandom (not implemented yet).
+
+Multiple seed sources can be specified in form of a comma separated list.
+Not all options are available on all platforms.
+The default option is --with-rand-seed=os, which is in general the best
+choice for the given operating system. So in general, it is not necessary
+to specify this option explicitely.
+
+Automatic seeding can be disabled entirely using the --with-rand-seed=none
+option. Some platforms have it disabled by default, because no suitable
+random seed source is available.
+
+The following two sections describe the reseeding process of the master
+DRBG, depending on whether automatic reseeding is configured or not.
+
+For more details about the configuration process, see the INSTALLATION
+instructions .
+A summary of the configure options can be displayed with the OpenSSL
+L<version(1)> command.
+
+
+=head2 Reseeding the master DRBG with automatic seeding enabled
+
+Calling RAND_poll() or RAND_add() is not necessary, because the DRBG
+pulls the necessary entropy from its source automatically.
+However, both calls are permitted, and do reseed the RNG.
+
+RAND_add() can be used to add both kinds of random input, depending on the
+value of the B<randomness> argument:
+
+=over 4
+
+=item randomness == 0:
+
+The random bytes are mixed as additional input into the current state of
+the DRBG.
+Mixing in additional input is not considered a full reseeding, hence the
+reseed counter is not reset.
+
+
+=item randomness > 0:
+
+The random bytes are used as entropy input for a full reseeding
+(resp. reinstantiation) if the DRBG is instantiated
+(resp. uninstantiated or in an error state).
+A reseeding requires 16 bytes (128 bits) of randomness.
```

Comment:
It requires 32 bytes now, you should probably say something that it depends on the strength of the DRBG.